specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/examples/LedgerDigestUploadEnable.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "ledgertestrg",
+    "serverName": "ledgertestserver",
+    "databaseName": "testdb",
+    "ledgerName": "current",
+    "api-version": "2021-02-01-preview",
+    "parameters": {
+      "properties": {
+        "digestStorageEndpoint": "https://MyAccount.blob.core.windows.net"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/ledgertestrg/providers/Microsoft.Sql/servers/ledgertestserver/databases/testdb/ledgerDigestUpload/current",
+        "name": "current",
+        "type": "Microsoft.Sql/servers/databases/ledgerDigestUpload",
+        "properties": {
+          "digestStorageEndpoint": "https://mystorage.blob.core.windows.net",
+          "state": "Enabled"
+        }
+      }
+    },
+    "202": {
+      "headers": {
+        "Location": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/ledgertestrg/providers/Microsoft.Sql/servers/ledgertestserver/databases/testdb/ledgerDigestUpload/current"
```

Comment:
This should be tracking an individual operation (i.e. the url should contain the operation id). See RPC: https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#202-accepted-and-location-headers

Reply:
fixed